src-vue/src/views/DeveloperView.vue

```diff
@@ -143,6 +157,34 @@ export default defineComponent({
                         position: 'bottom-right'
                     });
                 });
+        },
+        async installMod() {
+            let game_install = {
+                game_path: this.$store.state.game_path,
+                install_type: this.$store.state.install_type
+            } as GameInstall;
+            let mod_to_install = this.mod_to_install_field_string;
+            await invoke("install_mod_caller", { gameInstall: game_install, thunderstoreModString: mod_to_install }).then((message) => {
+                // Simply console logging for now
+                // In the future we should display the installed mods somewhere
+                console.log(message);
```

Comment:
This should be removed

Reply:
Done with 66007b91eb2fbe5805821ba23644a98586385f02